src/components/MetadataFields.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
+    <RecordMetadataField
+      v-if="title"
+      label="Title"
+      :values="title"
+      labelSuffix=":"
+    />
+    <RecordMetadataField
+      v-if="artist"
+      label="Artist"
+      :values="artist"
+      labelSuffix=":"
+    />
+    <div class="view-in-collection">
+      <a :href="collectionLink"
+        >View in the collection <Icon :name="'outbound'"
+      /></a>
+    </div>
+    <RecordMetadataField
+      v-if="attribution"
+      label="Attribution"
+      :values="attribution"
+      labelSuffix=":"
+    />
```

Comment:
```suggestion
     <RecordMetadataField		
                 v-if="searchResult.attribution"		
                 label="Attribution"		
                 labelSuffix=""		
                 ><span v-html="searchResult.attribution"></span		
               ></RecordMetadataField>
```
This got lost in the merge. I changed it to a v-html because the IIIF Manifest has a link in it

Reply:
@abrin oops, missed that! does the pushed change look right?